modules/gcp_bigquery_table/inputs.tf

```diff
@@ -28,3 +28,11 @@ variable "customer_managed_key_id" {
   type        = string
   description = "Describes the Cloud KMS encryption key that will be used to protect destination BigQuery table. The BigQuery Service Account associated with your project requires access to this encryption key."
 }
+
+variable "materialized_view" {
+  type = set(object({
+    query = string
+  }))
+  description = "create view for table"
```

Comment:
description not clear - what is this doing?

Reply:
I don't know what to explain more? it's a term in BQ where you can create view from a table https://cloud.google.com/bigquery/docs/views

Reply:
I think you can copy description and object description from https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#materialized_view and https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#query